content/en/docs/installation/kubernetes.md

```diff
@@ -29,19 +29,7 @@ guides](../../configuration/).
 
 ## Installing with regular manifests
 
-In order to install cert-manager, we must first create a namespace to run it in.
-This guide will install cert-manager into the `cert-manager` namespace. It is
-possible to run cert-manager in a different namespace, although you will need to
-make modifications to the deployment manifests.
-
-
-Create a namespace to run cert-manager in
-```bash
-$ kubectl create namespace cert-manager
-```
-
-We can now go ahead and install cert-manager. All resources
-(the `CustomResourceDefinitions`, cert-manager, and the webhook component)
+All resources(the `CustomResourceDefinitions`, cert-manager, namespace, and the webhook component)
```

Comment:
Can you add a space between the word `resources` and the bracketed bit? i.e. `All resources (the ...`